src/config/audiences.js

```diff
@@ -49,6 +49,16 @@ export default {
       host: 'https://facilities-staging.api.ndustrial.io'
     }
   },
+  files: {
+    production: {
+      clientId: '6jPAqJCMFMRP5t7Rw2X19qt2EtTfdQ2Q',
+      host: 'https://files.api.ndustrial.io'
+    },
+    staging: {
+      clientId: '6jPAqJCMFMRP5t7Rw2X19qt2EtTfdQ2Q',
+      host: 'https://files-staging.api.ndustrial.io'
```

Comment:
These client IDs are different now. And might change again in the future for reasons.

Current:
Staging: `SLE310LY4nDT5p06sUGPVOLlGXk5osbQ`
Production: `VZJ2MRd71dM9oX2hB1EVAEjUe6loL7pl`

Reply:
Ah, thanks. I'll update them to those for now.